bitnami/nginx-ingress-controller/templates/controller-service.yaml

```diff
@@ -74,21 +74,21 @@ spec:
       {{- end }}
     {{- end }}
     {{- range $key, $value := .Values.tcp }}
-    - name: {{ $key }}-tcp
+    - name: {{ if $.Values.portNamePrefix }}{{ $.Values.portNamePrefix }}-{{ end }}{{ $key }}-tcp
       port: {{ $key }}
       protocol: TCP
-      targetPort: {{ $key }}-tcp
+      targetPort: {{ if $.Values.portNamePrefix }}{{ $.Values.portNamePrefix }}-{{ end }}{{ $key }}-tcp
```

Comment:
We could maintain the `targetPort` and container port name in controller's deployment/daemonset unchanged right? I mean, the restrictions you're mentioning only affects the service ports' names

Reply:
Yes please, let's only apply the changes to service ports' names

Reply:
Done, please check it